extensions/juejin/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "jue-jin",
```

Comment:
I think we can use its Chinese name `掘金 juejin.cn` since your extension is Chinese only.

```suggestion
  "name": "掘金 juenjin.cn",
```

We can append its website to the name so users can search by its Pinyin name.

Reply:
We can't have Chinese in the name according to the manual; however, we can allow it in `keywords: []` instead

Reply:
Ok, then let's use `Juejin` as its name here. Dash is not needed.

Reply:
This is only the `name` field, which is not something the user will see anyway. They will only see the title 🙂

Reply:
Ah, then the lowercase `juejin` would be better.

Reply:
Okay, I've modified the `title` field to `Juejin` and added the keywords `Juejin - juejin.cn` to the `keywords` field. @pernielsentikaer Can I add `掘金` the Chinese name for this extension to `keywords`?